ExtensionExamples/ActionExtension/DocumentAction.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Intelledox.Extension.Action;
+using Intelledox.QAWizard;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SampleActionExtensions
+{
+    /// <summary>
+    /// This sample demonstrates dealing within generated documents that have been sent into the action.
```

Comment:
This sample demonstrates accessing and working with generated document binaries within an Action. The project’s administrator configures which document(s) should generated at the end of each workflow step and the final submission of the form.